### PR TITLE
Added Class for Deck and Cards

### PR DESCRIPTION
**Pull request by Tianster**
@cbas check out the use case for fat arrow functions :trollface:

### CODE REVIEW

File: dist/game.js [hunk 1]
```diff
@@ -27,36 +27,47 @@ var Card = (function () {
   return Card;
 })();
 
-var Deck = function Deck(rank, suits) {
```
Comment (cbas) [inline dist/game.js]: `.gitingore` the `dist/` folder

File: game/game.js [hunk 2]
```diff
@@ -4,46 +4,42 @@ var ranks = ['A', '2', '3', '4', '5',
...
+        this.cards.push(card)
+      })
+    })
+  }
+  displayDeck () {
+    this.cards.forEach(card => {
+      card.displayCard()
+    })
+  }
+  shuffleDeck () {
```
Comment (cbas) [inline game/game.js]: Check out the [Fisher–Yates shuffle](https://en.wikipedia.org/wiki/Fisher–Yates_shuffle) algorithm which is implemented in [lodash.shuffle()](https://lodash.com/docs#shuffle) and available on NPM as [lodash.shuffle](https://lodash.com/docs#shuffle).

**Comment by Tianster** [comment]
@cbas ; @sevresbabylone  please review 
stuck at how to compare deal card with previous card

**Comment by Tianster** [comment]
@cbas @sevresbabylone please review game logic .. will touch up on CSS and other event listeners

**Comment by Tianster** [comment]
@sevresbabylone pls comment

**Pull request merged**